superset-frontend/packages/superset-ui-chart-controls/src/types.ts

```diff
@@ -402,7 +402,8 @@ export type ConditionalFormattingConfig = {
   targetValueLeft?: number;
   targetValueRight?: number;
   column?: string;
-  colorScheme?: string;
+  colorScheme?: { r: number; g: number; b: number; a?: number };
```

Comment:
If we stick with the color picker, maybe this should be renamed. What do you want it to be?

Reply:
no longer applies